src/Security/Security/help/Get-AzSecurityPricing.md

```diff
@@ -29,40 +29,45 @@ Get-AzSecurityPricing -ResourceId <String> [-DefaultProfile <IAzureContextContai
 
 ## DESCRIPTION
 Azure Security Center pricing tier is decided per scope, with this cmdlet you can get the configured pricing tiers.
-Subscription pricing tier include all the resource groups under it.
-Resource Group pricing tier will override the subscription pricing tier.
```

Comment:
some comments 
- per scope in line 1 is not relevant either
- Suggest aligning with Pms to Azure Defender terminology
- Suggest adding explanantion as to Bundles
Azure Security Center pricing tier is decided per scope, with this cmdlet you can get the configured pricing tiers.

Suggest replacing deleted lines with comments about bundles, such as -

Reply:
@chshum, some comments:
1. In the UI we no longer refer to "bundles", it's all "plans".
1. It's a little weird to talk about "pricing tier" too, since that's also gone from the UI. 
1. There's a branding shift in which Azure Security Center is always free, and the charges are for Azure Defender bundles. 

What do you think of this intro replacing the current one?
**You can enable each Azure Defender plan, per subscription, using this cmdlet.**

Reply:
This cmdlet is "get", not "create", so it cant set anything. should be something like 
_"Gets all of the the Azure Defender plans configured for a subscription"_

Reply:
### Get-AzSecurityPricing
**SYNOPSIS**
Gets the pricing data for the Azure Defender plans enabled in Azure Security Center for a scope.

**DESCRIPTION**
Gets the status and pricing details of the Azure Defender plan for a subscription.
For details about Azure Defender and the available plans, see [Introduction to Azure Defender](https://docs.microsoft.com/azure/security-center/azure-defender)

### Set-AzSecurityPricing
**SYNOPSIS**
Sets the pricing data for the Azure Defender plans enabled in Azure Security Center for a scope.

**DESCRIPTION**
Enable or disable any of the Azure Defender plans for a subscription.
For details about Azure Defender and the available plans, see [Introduction to Azure Defender](https://docs.microsoft.com/azure/security-center/azure-defender).

Reply:
Is the `PricingTier` parameter still taking the value "Standard"?

### Example 1
```powershell
PS C:\> Set-AzSecurityPricing -Name "virtualmachines" -PricingTier "Standard"

Reply:
yes